tests/integration/helpers.py

```diff
@@ -60,3 +62,33 @@ async def get_unit_url(
         f"{application}/{unit}"
     ]["address"]
     return f"{protocol}://{address}:{port}"
+
+
+async def get_memberships(ops_test: OpsTest, url):
```

Comment:
Why only have type hints for one variable?

Reply:
No good reason, this happens throughout the charm code for ops_test, I will address this in a separate PR for all variables.